lib/dom.js

```diff
@@ -321,19 +339,31 @@ declare class Element extends Node {
     removeAttributeNS(namespaceURI: string, localName: string): void;
     querySelector(selector: string): HTMLElement;
     querySelectorAll(selector: string): NodeList<HTMLElement>;
+    getBoundingClientRect(): ClientRect;
+    insertAdjacentHTML(position: string, text: string): void;
+    blur(): void;
+    click(): void;
+    focus(): void;
```

Comment:
You are right, although the latest version of CSSOM has `getBoundingClientRect` on `Element`: http://dev.w3.org/csswg/cssom-view/#the-getclientrects()-and-getboundingclientrect()-methods I will move the other four. Should I move `getBoundingClientRect` as well?